packages/xo-web/src/xo-app/proxies/index.js

```diff
@@ -121,6 +122,15 @@ const COLUMNS = [
     name: _('name'),
     sortCriteria: 'name',
   },
+  {
+    // TODO: replace dummy URL by proxy.url
+    itemRenderer: ({ url }) => (
+      <Copiable tagName='pre' value='http://token@address'>
+        http://token@address
+      </Copiable>
+    ),
+    name: _('proxyUrl'),
+  },
```

Comment:
IMHO, it's not necessary to show the URL by default, it can be very long (due to the token) and is rarely useful.

I believe a button to copy to the clipboard in the advanced option would be enough :slightly_smiling_face: 

What do you think?